doc/api/tls.md

```diff
@@ -1048,6 +1079,20 @@ changes:
     verified against the list of supplied CAs. An `'error'` event is emitted if
     verification fails; `err.code` contains the OpenSSL error code. **Default:**
     `true`.
+  * `pskCallback` {Function}
+     * hint: {string} optional message sent from the server to help client
+       decide which identity to use during negotiation.
+     * Returns: {Object} in the form
```

Comment:
I don't see a way of doing that with the current OpenSSL api, perhaps we could somehow intercept the process during earlier stages of connection establishment so that at the point of PSK request we could provide the received result but I'm not that familiar with OpenSSL to do that.